.env.example

```
@@ -134,6 +134,9 @@ BREEZ_GREENLIGHT_INVITE_CODE=CODE
 BREEZ_GREENLIGHT_DEVICE_KEY="/path/to/breezsdk/device.pem"  # or BASE64/HEXSTRING
 BREEZ_GREENLIGHT_DEVICE_CERT="/path/to/breezsdk/device.crt"  # or BASE64/HEXSTRING
 
+# BreezLiquidSdkWallet
+BREEZ_MNEMONIC="MNEMONIC SEED PHRASE"
```

1. let's use BREEZ_LIQUID prefix to differentiate this from BreezSdkWallet
2. let's use seed instead of mnemonic, since BreezSdkWallet above uses this term
3. this has to be changed in `docs/guide/wallets.md` too and of course in the code

```suggestion
BREEZ_LIQUID_SEED=SEED
```